src/contexts/MediaQueryContext.tsx

```diff
@@ -0,0 +1,8 @@
+import { useMediaQuery } from 'react-responsive';
+
+const useIsMobile = () => {
+  const isMobile = useMediaQuery({ query: '(max-width: 640px)' });
```

Comment:
But if we want to change media query? I think we should pass the width as a props